lib/foreman/cron.rb

```diff
@@ -0,0 +1,63 @@
+require 'rake'
+
+module Foreman
+  module Cron
+    # This mirrors existing recurring jobs and can be extended in follow-up work.
+    # Plan is to later evolve this into a registry/plugin API.
+    TASKS_BY_CADENCE = {
```

Comment:
How would plugins enhance this list?

Reply:
@evgeni one thing i’m also wondering how you’d see logging/error handling work there – would we still centralize it in a helper (similar to `Foreman::Cron.run`) or just let the individual tasks log on their own?